### ISSUE TITLE
Data source for FR (fatalities is outdated)

### ISSUE THREAD

**Issue by mdeff** [daenuprobst/covid19-cases-switzerland#38]
It's a pity that the canton doesn't publish official numbers on <https://www.fr.ch/de/covid19/gesundheit/covid-19/coronavirus-entwicklungen-der-situation> anymore. Where does the current data come from?

I don't know if the number of cases is correct, but the number of fatalities is off. While the data in this repository shows 1 death, at least 4 deaths were reported by the local medias as of 2020-03-22 [[La Liberté](https://www.laliberte.ch/info-regionale/sante/en-continu-un-4e-deces-dans-le-canton-de-fribourg-558068), [Radio Fr](https://www.radiofr.ch/fribourg/info/news/2020/3/21/deux-nouveaux-deces-du-coronavirus-a-fribourg.html)].

**Comment by grssnbchr** [comment]
I would also be interested in that. Where does the data come from?

**Comment by BARMA2** [comment]
I think they don’t count fatalities of people died with co-morbidity...

**Comment by mdeff** [comment]
I don't think so. From a source at the [FR hospital](https://www.h-fr.ch), the COVID-19 fatalities are announced to the Federal Office of Public Health. ([The procedure and notification forms are public.](https://www.bag.admin.ch/infreporting)) So they are probably counted at the national level. The only problem is to find an official public source for the canton. That is a known problem, also discussed in <https://github.com/openZH/covid_19/issues/129> with a workaround through BAG in <https://github.com/openZH/covid_19/issues/210>.

**Comment by jemmail** [comment]
Here is the official data for Fribourg.. Please update

https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
